crates/extensions/tedge_mqtt_ext/Cargo.toml

```diff
@@ -20,6 +20,7 @@ async-trait = { workspace = true }
 mqtt_channel = { workspace = true }
 serde_json = { workspace = true }
 tedge_actors = { workspace = true }
+tedge_api = { workspace = true }
```

Comment:
This dependency raises concerns.

`tedge` is more specific than `mqtt`. Hence, the latter should not depend on the former.